Makefile

```diff
@@ -257,6 +257,22 @@ clean-for-device-build:
 .PHONY: clean-test-with-device-farm
 clean-test-with-device-farm: clean-for-device-build test-with-device-farm
 
+.PHONY: test-with-firebase
+test-with-firebase:
+	xcodebuild \
+		-workspace Apps/Apps.xcworkspace \
+		-scheme $(SCHEME) \
+		-enableCodeCoverage YES \
+		-configuration $(CONFIGURATION) \
+		-sdk iphoneos build-for-testing;
```

Comment:
It looks like we have double build-for-testing logic – first here and the second in Fastfile. Do we need them both?

Reply:
Removed this part.